src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/TestHost.cs

```diff
@@ -93,9 +136,14 @@ public byte[] GetPeerCert(bool exportAsPem)
             }
         }
 
-        public string GetFQDN()
+        public Stream Ping()
```

Comment:
Can this just be a bool?  What is the advantage of using a message as a stream?  Is it easier or harder to consume this response from curl or a browser?